scenario/listboxselect.go

```diff
@@ -69,7 +72,7 @@ func (value ListBoxSelectType) MarshalJSON() ([]byte, error) {
 
 // String representation of ListBoxSelectType
 func (value ListBoxSelectType) String() string {
-	sType, err := listBoxSelectTypeEnumMap.String(int(value))
+	sType, err := value.GetEnumMap().String(int(value))
```

Comment:
I would prefer if we didn't allocate a new object for each user each time this is executed since it never changes anyway.
you could keep the listBoxSelectTypeEnumMap you had before and just return this in the GetEnumMap function.

Reply:
I'm with you on that! It will not be optimized just because it could be optimized ;)